packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -1296,6 +1296,16 @@ ensureIndex(Posts,
   { name: "posts.recommendable" }
 );
 
+Posts.addView("hasEverDialogued", (terms: PostsViewTerms) => {
+  return {
+    selector: {
+      $or: [{userId: terms.userId},
+        {"coauthorStatuses.userId": terms.userId}],
```

Comment:
formatting here is weird; either keep it all on one line or put enclosing brackets on their own lines:
```
$or: [
  { ... },
  { ... }
]
```